pkg/ebpf/c/tracee.bpf.c

```diff
@@ -783,10 +783,16 @@ typedef struct task_info {
     u8 container_state;   // the state of the container the task resides in
 } task_info_t;
 
+typedef struct binary {
+    u64 mnt_id;
```

Comment:
On my machine the max mnt_id is `127292` (from `/proc/sys/user/max_mnt_namespaces`).
If this is uniform across systems, then u64 max is much larger.
I wonder if we can use int64 here instead, then when `no_mnt_id=1` we set `mnt_id=-1`.
Then we could later use only one map lookup:
`equality = bpf_map_lookup_elem(&binary_filter, &proc_info->binary);`
Which would compare both the binary and the mnt_id which in a "neutral mode" would always be -1.
Could this work?

Reply:
ns_id is 32 bit indeed. I will change it. Anyway, no_mnt_id is always zero. What do you suggest exactly?

Reply:
changed to u32